packages/web3-providers-ws/src/index.ts

```diff
@@ -80,14 +80,14 @@ export default class WebSocketProvider<
 		this._clientUrl = clientUrl;
 		this._wsProviderOptions = wsProviderOptions;
 
-		const DEFAULT_WS_PROVIDER_OPTIONS = {
+		const DEFAULT_RECON_WS_PROVIDER_OPTIONS = {
```

Comment:
I think it's the project preference to not abbreviate things, I'd suggest renaming this to `DEFAULT_PROVIDER_RECONNECTION_OPTIONS`